src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -766,35 +788,58 @@ export class StoreQuickstartSetup extends SfdxCommand {
                 )
             );
         }
-        if (!fs.existsSync(`${this.storeDir}/experience-bundle-package/unpackaged/experiences`)) {
+        if (!fs.existsSync(`${this.storeDir}/experience-bundle-package/unpackaged`)) {
             await this.statusFileManager.setValue('retrievedPackages', false);
             await this.retrievePackages();
         }
         this.ux.log(msgs.getMessage('quickstart.setup.makeSiteAndNavMenuItemPublic'));
-        const siteConfigMetaFileName =
-            this.storeDir +
-            `/experience-bundle-package/unpackaged/experiences/${
-                this.varargs['communityExperienceBundleName'] as string
-            }/config/${configName}.json`;
-        const siteConfigMetaFile = Object.assign(new StoreConfig(), await fs.readJson(siteConfigMetaFileName));
-        siteConfigMetaFile.isAvailableToGuests = true;
-        siteConfigMetaFile.authenticationType = 'AUTHENTICATED_WITH_PUBLIC_ACCESS_ENABLED';
-        fs.writeFileSync(siteConfigMetaFileName, JSON.stringify(siteConfigMetaFile, null, 4));
-        const def = parseStoreScratchDef(this.flags);
-        const relaxedLevel = def.settings.isRelaxedCSPLevel;
-        const siteConfigMainAppPageFileName =
-            this.storeDir +
-            `/experience-bundle-package/unpackaged/experiences/${
-                this.varargs['communityExperienceBundleName'] as string
-            }/config/mainAppPage.json`;
-        if (relaxedLevel) {
-            fs.writeFileSync(
-                siteConfigMainAppPageFileName,
-                fs
-                    .readFileSync(siteConfigMainAppPageFileName)
-                    .toString()
-                    .replace('"isRelaxedCSPLevel" : false,', '"isRelaxedCSPLevel" : true,')
-            );
+        let siteConfigMetaFileName = null;
+        let bundleName = this.varargs['communityExperienceBundleName'] as string;
+        if (this.isUsingDigitalExperienceBundle) {
```

Comment:
There is a lot of repeated code between the two if statement branches. Could you think through some options to create a more generic code block, maybe with some parameters to reduce the amount of duplicated code?

Reply:
I can see what I can do here. Thank you for your suggestion!